csharp-rest-service/Controllers/CustomerProfilesController.cs

```diff
@@ -0,0 +1,105 @@
+using Microsoft.AspNetCore.Mvc;
+using Microsoft.EntityFrameworkCore;
+using RestService.Models;
+
+namespace RestService.Controllers;
+
+[Route("api/customer-profiles")]
+[ApiController]
+public class CustomerProfilesController : ControllerBase
+{
+    private readonly CustomerProfilesContext _context;
+
+    private readonly ILogger<CustomerProfilesController> _logger;
+
+    public CustomerProfilesController(CustomerProfilesContext context, ILogger<CustomerProfilesController> logger)
+    {
+        _context = context;
+        _logger = logger;
+    }
+
+    // GET: api/customer-profiles
```

Comment:
I think you can remove the "// METHOD: path" comments.  You're effectively repeating the [Route] and [HttpMETHOD] attributes and not adding more detail.

Reply:
I agree they aren't _needed_ and could be inferred, but I left them because they're from a template and I've seen these accelerators used relatively frequently used by people that don't always know ASP.NET Core conventions so it may not be clear. I don't feel extremely strongly though, so if you still think I should remove them I can do that